Bottom: 9d4c5d8f580b87be2ed470a5115a4d4f6d956bc6
Top:    9dd2e8333309c3245ae1e8b79fa429c2a0d3fd5f
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-10 09:43:54 +0800

Refresh of autofs-5.0.6-fix-compilation-of-lookup_ldap_c-without-sasl.patch

---

diff --git a/modules/Makefile b/modules/Makefile
index 36e31e1..2589ae0 100644
--- a/modules/Makefile
+++ b/modules/Makefile
@@ -49,7 +49,7 @@ ifeq ($(LDAP), 1)
   LIBLDAP += $(XML_LIBS)
   ifeq ($(SASL), 1)
     SASL_OBJ = cyrus-sasl.o cyrus-sasl-extern.o
-    LDAP_FLAGS += $(SASL_FLAGS) $(KRB5_FLAGS) -DLDAP_THREAD_SAFE
+    LDAP_FLAGS += $(SASL_FLAGS) $(KRB5_FLAGS)
     LIBLDAP += $(LIBSASL) $(KRB5_LIBS)
   endif
 endif
